### ISSUE TITLE
 [CONVEX A(openai:generateAudioAction)] Error: 429 You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs:

### ISSUE THREAD

**Issue by Mkaell** [adrianhajdin/podcastr#7]
i have the such message, can i use gpt api on free account?
becuse i dont have request in dashboard
![image](https://github.com/adrianhajdin/podcastr/assets/81435704/08f3c909-427e-4d7a-806c-1ef6e013d9ad)

**Comment by AdityaSaini06** [comment]
I have the same doubt , did you find any solution?

**Comment by aadityesh** [comment]
I have the same issue. Upon visiting the OpenAI Dashboard, It clearly says that I have $5 limit (0% consumed) and I have made 0 calls to the API.

**Comment by Mikssxed** [comment]
Same problem here, any fixes?

**Comment by rado-kolev** [comment]
I found something here...
https://community.openai.com/t/api-openai-generateaudioaction-error-429-you-exceeded-your-current-quota-please-check-your-plan-and-billing-details/824282

**Comment by PhatDoge** [comment]
yeah i had the same issue, put 5 bucks on openai and immediately fixed
